### DISCUSSION TITLE
Which version contains the Skipped Test Enhancement referenced in closed issue 967

### DISCUSSION THREAD

**Discussion by RichFlickstein** [reportportal/reportportal#2066] [General]
We are currently using Report Portal V 5.5.1 but will be updating soon.
Currently, tests with the status of Skipped show as To Investigate in RP GUI. It seems that this behavior will be changed in RP 6.x according to what I saw in the following ticket. 
https://github.com/reportportal/reportportal/issues/1967

When I look to see the latest Report Portal, I see the versioning went from 5.x to version 23.1. Is that a versioning error or is it 5.23.1? Or is it that this version is the 6x but you changed your versioning? Really what I am looking for is to get the change referenced in 6.0.

**Comment by DzmitryHumianiuk** [comment]
Certainly! Here's a revised version:

@RichFlickstein Hi,

First of all, we've changed our version naming from semantic versioning (sem-ver) to a calendar-based approach. You can find the details here: https://reportportal.io/blog/new-approach-to-versions-naming

So, 23.1 is the correct version, as it represents the next incremental release. We'll announce separately when the major changes, codenamed RPv6, will come into play. Most likely, they'll start to come incrementally, since a complete stop of v5.x and switch to a full refactor would lead to a non-release phase for over a year.

Secondly, based on this issue: https://github.com/reportportal/reportportal/issues/1967, people were asking how to avoid marking Skipped issues as Defected by default. The answer lies in the configuration parameters for the launch. In most cases, we have a configuration property file with something like `rp.skipped.issue = true`.

In the next releases (where I'm referring to RPv6), we will get rid of the `NOT_ISSUE` object and replace it with the correct form.
